openmc/lattice.py

```diff
@@ -950,26 +957,24 @@ def universes(self, universes):
 
         # Check to see if the given universes look like a 2D or a 3D array.
         if isinstance(self._universes[0][0], openmc.Universe):
-            n_dims = 2
-
+            pass
         elif isinstance(self._universes[0][0][0], openmc.Universe):
-            n_dims = 3
-
+            pass
```

Comment:
Correct me if I'm wrong -- this entire `if` block looks superfluous since the call the `cv.check_iterable_type` above will achieve the same purpose.